ui/__tests__/components/contact-email.test.js

```diff
@@ -0,0 +1,30 @@
+import { mount } from 'enzyme';
+import React from 'react';
+
+import { email, ContactEmail } from '../../components/contact-email';
+
+const expr = `a[href="mailto:${email}"]`;
+
+describe('<ContactEmail />', () => {
+  describe('when given no props', () => {
+    const mailtext = ContactEmail.defaultProps.text;
```

Comment:
Mind adding a quick assert that this is non-empty? Right now we're asserting against a variable, which could be unintentionally empty and throw off the test.

Reply:
👍